m4/odp_libconfig.m4

```diff
@@ -5,7 +5,8 @@ AC_DEFUN([ODP_LIBCONFIG],
 ##########################################################################
 # Check for libconfig availability
 ##########################################################################
-PKG_CHECK_MODULES([LIBCONFIG], [libconfig])
+PKG_CHECK_MODULES([LIBCONFIG], [libconfig], [have_libconfig=yes])
+AM_CONDITIONAL([LIBCONFIG], [test x$have_libconfig = xyes])
```

Comment:
Would it be better to have this commit as the first commit so that build without libconfig present does not break in between?

Reply:
Sure, v3.